rules/list-item.js

```diff
@@ -81,6 +81,36 @@ function findAllLists(ast) {
 	return lists;
 }
 
+function findSublists(node) {
```

Comment:
I'd prefer to use https://github.com/syntax-tree/unist-util-visit instead of rewriting the recursive visitor pattern here.

Reply:
For some reason this didn't work at first, but now it works as expected 🤷🏻‍♂️